cachito/web/api_v1.py

```diff
@@ -65,29 +64,21 @@ def download_archive(request_id):
         raise ValidationError(
             'The request must be in the "complete" state before downloading the archive')
 
-    cachito_shared_dir = flask.current_app.config['CACHITO_SHARED_DIR']
-    wait_timeout = flask.current_app.config['CACHITO_WAIT_TIMEOUT']
-
-    with tempfile.TemporaryDirectory(prefix='cachito-', dir=cachito_shared_dir) as temp_dir:
-        # Although the cachito_shared_dir volume is required to be the same between celery
-        # workers and the API, they may be mounted at different locations. Use relative
-        # paths to agree on data location within the shared volume.
-        relative_temp_dir = os.path.basename(temp_dir)
-        relative_deps_path = os.path.join(relative_temp_dir, 'deps')
-        relative_bundle_archive_path = os.path.join(relative_temp_dir, 'bundle.tar.gz')
-        absolute_bundle_archive_path = os.path.join(
-            cachito_shared_dir, relative_bundle_archive_path)
-
-        # Chain tasks
-        chain_result = chain(
```

Comment:
I think I don't know the code well enough to understand how these three tasks are replaced by single `add_deps_to_bundle_archive` call run on backend. I only presume that those tasks move somehow files from backend to frontend, so the frontend code can repackage them into bundle.

Reply:
The previous code to get the download bundle was basically duplicating a lot of tasks that happen as part of the request handling. This is no longer needed since the bundle is created as part of the request.